runtime/Cargo.toml

```diff
@@ -71,6 +71,8 @@ solana-transaction-status = { workspace = true }
 solana-version = { workspace = true }
 solana-vote = { workspace = true }
 solana-vote-program = { workspace = true }
+solana-zk-elgamal-proof-program = { workspace = true }
+solana-zk-sdk = { workspace = true }
```

Comment:
nit: Sorry I'm only just noticing this now, but it seems like we only need this dependency for the program id -- can we also have it in `solana-zk-elgamal-proof-program` and remove this direct dependency? This can be done in a follow-up